formazione/templates/aspirante_corso_base_scheda.html

```diff
@@ -153,7 +153,12 @@ <h4><i class="fa fa-fw fa-warning"></i> Genera il verbale e termina il corso</h4
                     Scarica report
                 </a>
             </li>
-
+            <li role="presentation" {% if corso.url_firme in request.path %}class="active"{% endif %}>
+                <a href="{{ corso.url_firme }}" class="text-danger">
```

Comment:
Perche' `text-danger`? Non e' un elemento di errore.

Reply:
è uguale in scarica report

Reply:
Mi aspetterei `text-danger` solo per qualcosa che richiede attenzione da parte dell'utente, ad esempio un documento che puo' essere generato una sola volta. In entrambi i casi non credo sia necessario. Non so perche' sia cosi' per il report.